README.md

```diff
@@ -3,6 +3,10 @@ android-apidemos
 
 A fork of Google's Android ApiDemos application, used for testing Appium
 
+#### Prerequisites
+
+- Java 17
```

Comment:
npm/node are also needed to run the build

Reply:
done